testsuite/features/secondary/srv_restart.feature

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Restart the spacewalk-services via UI
```

Comment:
`Restart the spacewak services via UI`

(no dash)

Reply:
Fixed